Workbooks/Azure Monitor - Agents/Health Service Performance/settings.json

```diff
@@ -1,6 +1,6 @@
 {
     "$schema": "https://github.com/Microsoft/Application-Insights-Workbooks/blob/master/schema/settings.json",
-    "name":"Log Analytics Agent Health - Health Service",
+    "name":"Health Service",
```

Comment:
To be consistent, why not "Health Service Performance" like you did for the workbook focused on Monitoring Host?